engines/pytorch/pytorch-engine/src/main/java/ai/djl/pytorch/engine/PtNDArrayEx.java

```diff
@@ -694,7 +696,59 @@ public NDList multiBoxPrior(
             List<Float> steps,
             List<Float> offsets,
             boolean clip) {
-        throw new UnsupportedOperationException("Not implemented");
+        //Based on https://github.com/apache/mxnet/blob/a720b15b5fa011a9610dfaeabf0792443c6abec5/src/operator/contrib/multibox_prior.cc
+
+        NDManager ndManager = array.getManager().getParentManager();
+
+        Float step_x   = steps.get(1);
+        Float step_y   = steps.get(0);
+        int num_sizes  = sizes.size();
+        int num_ratios = ratios.size();
+        int count = 0;
+        float[][] out = new float[num_sizes][num_ratios];
```

Comment:
the out dimension is wrong, please make sure your implementation is correct

Reply:
Agree this is wrong. Comes out very clearly in the unit/integration test: `MultiBoxPriorTest`. I am asking for help. 
The code here
https://github.com/apache/mxnet/blob/master/src/operator/contrib/multibox_prior.cc#L53
refers to a two dimensional array. 

In addition, `manager.create` takes a two dimensional array. 

Can you help me out with where in the process you add the third dimension?

Reply:
Thanks for the help.